docs/setting_up_your_development_environment.md

```diff
@@ -1,42 +1,40 @@
 Setting up Your Hypothesis Development Environment
 ==================================================
 
-Set up your GitHub account and SSH key
---------------------------------------
+This guide will walk you through setting up your Hypothesis development environment: installing everything and getting the apps running locally.
```

Comment:
Always good to start a document with a sentence explaining what the document is/does.